src/utils/getImageId.ts

```diff
@@ -1,16 +1,23 @@
 import { shell } from "./shell";
+const Docker = require('dockerode')
 
+const docker = new Docker();
+
+export async function getImageId(imageTag: string): Promise<string> {
+  const image = docker.getImage(imageTag)
+  return await image.inspect().Id.replace(/['"]+/g, "")
+}
 /**
  * Returns the formated ID of a docker image
  *
  * @param imageTag i.e. admin.dnp.dappnode.eth:0.1.14
  * @param shell dependency
  * @return formated ID: sha256:0d31e5521ef6e92a0efb6110024da8a3517daac4b1e4bbbccaf063ce96641b1b
  */
-export async function getImageId(imageTag: string): Promise<string> {
+/* export async function getImageId(imageTag: string): Promise<string> {
```

Comment:
You should re-use this [function JSDocs](https://jsdoc.app/about-getting-started.html) and you can delete the old code.